src/screens/Analytics/PaymentsAnalytics/PaymentAnalytics.res

```diff
@@ -243,11 +245,14 @@ let make = () => {
     <div className="flex flex-col gap-5">
       <div className="flex items-center justify-between ">
         <PageUtils.PageHeading title />
-        <OMPSwitchHelper.OMPViews
-          views={OMPSwitchUtils.analyticsViewList(~checkUserEntity)}
-          selectedEntity={analyticsEntity}
-          onChange={updateAnalytcisEntity}
-        />
+        <Portal to="PaymentAnalyticsOMPView">
```

Comment:
why Portal is required over here

Reply:
It is required to show OMP views in correct place in Analytics section, please check UI once